lib/tpl/dokuwiki/tpl.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace dokuwiki\template\dokuwiki;
+
+/**
+ * Class tpl
+ *
+ * Provides additional template functions for the dokuwiki template
+ * @package dokuwiki\tpl\dokuwiki
+ */
+class tpl {
+
+    static $icons = array(
+        'default' => '00-default_checkbox-blank-circle-outline.svg',
+        'edit' => '01-edit_pencil.svg',
+        'create' => '02-create_pencil.svg',
+        'draft' => '03-draft_android-studio.svg',
+        'show' => '04-show_file-document.svg',
+        'source' => '05-source_file-xml.svg',
+        'revert' => '06-revert_replay.svg',
+        'revs' => '07-revisions_history.svg',
+        'backlink' => '08-backlink_link-variant.svg',
+        'subscribe' => '09-subscribe_email-outline.svg',
+        'top' => '10-top_arrow-up.svg',
+        'mediaManager' => '11-mediamanager_folder-image.svg',
+        'img_backto' => '12-back_arrow-left.svg',
+
+    );
+
+    /**
+     * Return the HTML for one of the default actions
+     *
+     * Reimplements parts of tpl_actionlink
+     *
+     * @param string $action
+     * @return string
+     */
+    static public function pageToolAction($action) {
+        $data = tpl_get_action($action);
+        if(!is_array($data)) return '';
+        global $lang;
+
+        if($data['id'][0] == '#') {
+            $linktarget = $data['id'];
+        } else {
+            $linktarget = wl($data['id'], $data['params']);
```

Comment:
We want `&` as the parameter separator here. Using the default `&amp;` seems to break the link, at least on my machine.